pronto_quadruped/include/pronto_quadruped/LegOdometer.hpp

```diff
@@ -109,13 +109,6 @@ class LegOdometer  : public LegOdometerBase {
         std::cerr << "[LegOdometer::getOrientation] Function not implemented yet!" << std::endl;
     }
 
-    /**
```

Comment:
where did this function go?

Reply:
I searched the codebase and it wasn't used anywhere currently in this repository nor in the robot-specific ones. Should I add it back? The GRF Delta is explicitly computed when the GRF is set.